pkg/internal/config_resolver.go

```diff
@@ -11,24 +11,24 @@ import (
 )
 
 type ConfigMatch struct {
-	OriginalMatch            *prefabProto.ConfigValue
-	Match                    *prefabProto.ConfigValue
-	SelectedConditionalValue *prefabProto.ConditionalValue
-	OriginalKey              string
-	WeightedValueIndex       int
-	RowIndex                 int
-	ConditionalValueIndex    int
-	IsMatch                  bool
+	ConfigID              int64
+	ConfigType            prefabProto.ConfigType
+	OriginalMatch         *prefabProto.ConfigValue
+	Match                 *prefabProto.ConfigValue
+	ConfigKey             string
+	WeightedValueIndex    *int
+	RowIndex              *int
+	ConditionalValueIndex *int
+	IsMatch               bool
```

Comment:
These changes aren't exciting, they just reflect the change in needs between expectation/anticipation and implementation